articles-1.html

```diff
@@ -54,24 +54,26 @@ <h1>All Articles</h1>
 					<p>Learn what some of the best products are out there for clearing acne</p>
 				</div>
 
-				<p class="subheading">Read articles about oily skin issues</p>
+				<p class="subheading">Read articles dedicated to oily skin</p>
 
 				<div class="image-first">
-					<a href="#"><img src="sample.png"></a>
+					<a href="clear-oily-skin.html"><img src="images/clear-oily-skin.jpg"></a>

```

Comment:
I included a completed article that users can click on and an image to go with it.